charts/kube-prometheus-stack-bootstrap/templates/alertmanagerconfig.yaml

```diff
@@ -180,6 +180,16 @@ spec:
       {{- include "slack.template" . | nindent 6 }}
       text: |-
         {{- include "slack.text" . | nindent 8 }}
+  inhibitRules:
+  - sourceMatch:
```

Comment:
In the docs linked to in the description, the syntax looks different. ie source_match etc. Do you know why it's camel case here? Presume camel case is correct, as that matches the rest of the keys in this file, just not sure I understand why!